pipert/contrib/database/handler.py

```diff
@@ -0,0 +1,176 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc as sql_exc, Column, Integer, ARRAY, Float, JSON, Text, DateTime
+import re
+
+
+class DBHandler:
```

Comment:
Is this class only for Postgresql? If so, I suggested that the name of the class will be PSQLDBHandler.